composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "weebly/cloud-client",
+    "name": "amarsyla/cloud-client",
```

Comment:
If you'd like us to merge this PR, you'll need to change the package name back to `weebly/cloud-client`.